src/app/api/getuser/route.ts

```diff
@@ -0,0 +1,13 @@
+import {  NextResponse } from "next/server";
+import User from "@/models/userModel";
+import connection from "@/db/config";
+
+export async function GET(){
```

Comment:
why calling connection again and again. you should create db connection once  . not all the time, if you create a lot of connection it will be overload on the db and decrease performance